test/cypress/support/commands.js

```diff
@@ -151,6 +151,11 @@ Cypress.Commands.add('giCreateGroup', (name, {
 
     cy.url().should('eq', 'http://localhost:8000/app/dashboard')
     cy.getByDT('groupName').should('contain', name)
+    // wait for general chatroom contract to finish syncing
+    cy.wait(500) // eslint-disable-line
```

Comment:
Same comment as above.

Reply:
This `cy.wait` waits for `General` channel is synced. Just added a function which checks if `General` channel is synced and replaced. But will remove this `cy.wait` a bit later after testing more and more.

Reply:
Done. Just go to the Group Chat page and check if the `General` chatroom is fully synced.